docs/resources/apig_throttling_policy.md

```diff
@@ -0,0 +1,142 @@
+---
+subcategory: "API Gateway (APIG)"
+---
+
+# huaweicloud_apig_throttling_policy
+
+Manages an APIG (API) throttling policy resource within HuaweiCloud.
+
+## Example Usage
+
+### Create a basic throttling policy
+```hcl
+variable "instance_id" {}
+variable "policy_name" {}
+variable "description" {}
+
+resource "huaweicloud_apig_throttling_policy" "test" {
+  instance_id       = var.instance_id
+  name              = var.policy_name
+  description       = var.description
+  type              = "API-based"
+  period            = 10
+  period_unit       = "MINUTE"
+  max_api_requests  = 70
+  max_user_requests = 45
+  max_app_requests  = 45
+  max_ip_requests   = 45
+}
+```
+
+### Create a throttling policy with a special throttle
+```hcl
+variable "instance_id" {}
+variable "policy_name" {}
+variable "description" {}
+variable "application_id" {}
+
+resource "huaweicloud_apig_throttling_policy" "test" {
+  instance_id       = var.instance_id
+  name              = var.policy_name
+  description       = var.description
+  type              = "API-based"
+  period            = 10
+  period_unit       = "MINUTE"
+  max_api_requests  = 70
+  max_user_requests = 45
+  max_app_requests  = 45
+  max_ip_requests   = 45
+
+  app_throttles {
+    max_api_requests     = 40
+    throttling_object_id = var.application_id
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the API throttling policy resource.
+  If omitted, the provider-level region will be used.
+  Changing this will create a new API throttling policy resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies an ID of the APIG dedicated instance to which the
+  API throttling policy belongs to.
+  Changing this will create a new API throttling policy resource.
+
+* `name` - (Required, String) Specifies the name of the API throttling policy.
+  The policy name consists of 3 to 64 characters, starting with a letter.
+  Only letters, digits and underscores (_) are allowed.
+
+* `period` - (Required, Int) Specifies the period of time for limiting the number of API calls.
+  This parameter applies with each of the API call limits: `max_api_requests`, `max_app_requests`, `max_ip_requests`
+  and `max_user_requests`.
+
+* `max_api_requests` - (Required, Int) Specifies the maximum number of times an API can be accessed within a specified
```

Comment:
please put  `max_api_requests`, `max_app_requests`, `max_ip_requests` and `max_user_requests` together.